Assets/Scripts/Audio/FMODDestroy.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FMODDestroy : MonoBehaviour
```

Comment:
What does this do??
Also just remove the default functions + comments below if you're not using them!